src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService us=new UserService();
+        if(us.findByEmail(email)!=null && us.findByEmail(email).getPassword()==password){
+            return true;
+        }else {
+            return false;
```

Comment:
```suggestion
        return us.findByEmail(email) != null && us.findByEmail(email).getPassword() == password;
```
you can simplify  this code